CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## Master
+
+#### Enhancements
+
+* None
+
+#### Bug Fixes
+
+* None
+
+## 0.1.0 (2021-01-13)
```

Comment:
Just FYI, I don't see a release created in GitHub yet. We'll probably need to circle back and update this once the initial release is actually published.